docs/management/watcher-ui/watcher-getting-started.asciidoc

```diff
@@ -0,0 +1,40 @@
+
+[[watcher-getting-started]]
+=== View watch history and status
+The Watcher overview page lists all your watches,
+including their state: `Firing`, `Error`, `OK`, or `Disabled`.
```

Comment:
Maybe not for this PR, but it would be cool if these states were defined. For example, I'm not 100% sure what `Firing` means.